modules/profile/manifests/st2server.pp

```diff
@@ -222,6 +222,14 @@
     groups  +> $_st2_packs_group,
   }
 
+  file { '/opt/stackstorm/packs':
+    ensure  => 'directory',
+    owner   => 'root',
+    group   => $_st2_packs_group,
+    mode    => 0775,
```

Comment:
Error: Parameter mode failed on File[/opt/stackstorm/packs]: The file mode specification must be a string, not 'Fixnum' at /opt/puppet/environments/packs_perms/modules/profile/manifests/st2server.pp:225